opendcs-rest-api-jetty/build.gradle

```diff
@@ -46,15 +44,19 @@ task startJetty(type: JavaExec) { ->
             "-s")
 }
 
-task prepareStartScript(type: Copy) {
-    from './bin/start.sh'
-    into "${buildDir}/odcsapi-jetty-standalone/bin/"
-    filter(ReplaceTokens, tokens: ['jarName': jar.archiveFileName.get()])
+task createStartScript() {
+    doLast {
+        new File("${buildDir}/bin", "start.sh").text = """
+#!/bin/bash
+cd `dirname \$0`/..
+java \$JAVA_OPTS -jar libs/${project.name}-${project.version}.jar \$JAVA_ARGS
```

Comment:
There appears to be an issue in the chain of tasks. The action build is showing start.sh as missing.

Reply:
It's fixed on my end, going to do a new push soon.